README.rst

```diff
@@ -19,69 +19,72 @@ developed at CEGeME.
    <a href="http://musica.ufmg.br/cegeme" target="_blank">CEGeME</a>
 
 
-**********
-Installing
-**********
+************
+Installation
+************
+
+To install iracema, simply run:
+
+.. code-block:: bash
 
-Linux
-=====
+   pip3 install iracema
 
-1. Clone the repository from github:
 
+If you're a Linux user, you will need to manually install an audio I/O library called PortAudio. This is already installed by default in MacOS. In Debian / Ubuntu you can install it using apt:
 
 .. code-block:: bash
 
-   git clone --recurse-submodules https://github.com/cegeme/iracema.git
+   sudo apt install libportaudio2
 
 
-The command shown above will clone the project including some example audio
-files. If you don't want to download those files, you should omit the
-parameter ``--recurse-submodules``, like this:
+Note that we strongly recommend that you create a |virtual environment| to install the dependencies for iracema, since it is always a good practice to keep project-specific dependencies isolated from your base Python installation.
```

Comment:
Check this: https://docutils.sourceforge.io/docs/user/rst/quickref.html#hyperlink-targets